example/conduit/t8forest_to_conduit_mesh.cxx

```diff
@@ -0,0 +1,429 @@
+#include <t8_forest_vtk.h>
+#include <t8_vtk.h>
+#include <t8_cmesh.h>
+#include <t8_element_cxx.hxx>
+#include <t8_forest/t8_forest_ghost.h>
+#include <t8_vec.h>
+#include "t8_cmesh/t8_cmesh_trees.h"
+
+#include <t8_cmesh/t8_cmesh_examples.h>
+#include <t8_forest.h>
+#include <t8_schemes/t8_default_cxx.hxx>
+#include <t8_forest_vtk.h>
+
+#include <t8.h>
+#include <t8_forest.h>
+#include <t8_schemes/t8_default_cxx.hxx>
+
+/*#include <catalyst.hpp>*/
+
+#include "conduit.hpp"
+#include "conduit_relay.hpp"
+#include "conduit_blueprint.hpp"
+
+#include <t8_forest_catalyst.hxx>
+
+using namespace     conduit;
+
+/*
+ * Funktion die t8code Gitter in ein conduit mesh blueprint konformes Gitter übersetzt. 
+ */
+
+Node
```

Comment:
This function should be moved into a seperate file, since we want to use it to write meshes.